### PR TITLE
fix: update getHarnessEditContentsUrl

### PR DESCRIPTION
**Pull request by cjlee01**
## Hey, I just made a Pull Request!

Updated function for getHarnessEditContentsUrl as it was missing 'repos'

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/dull-moose-cough.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': minor
```
Comment (freben) [inline .changeset/dull-moose-cough.md:2]:
```suggestion
'@backstage/integration': patch
```

sort of goes on top of other things that are going on here so it's fine

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
yeah the approval seems a bit confusing, i wanted to auto-apply that suggestion but apparently github won't let me - feel free to change it to patch yourself and i'll merge when that's done

**Comment by freben** [comment]
@cjlee01 @OrkoHunter  just a gentle ping so you get this into the release on Tuesday.

**Comment by OrkoHunter** [comment]
@freben Applied your suggestion! I guess since this PR is coming from a fork, it's not letting you commit. And thanks for the reminder 🙏

**Review by freben — APPROVED** [review]

**Comment by awanlin** [comment]
Hi @OrkoHunter, I'm a little late here but wanted to mention that Harness looks to have support for `edit_url` in TechDocs but it's not enabled here:

https://github.com/backstage/backstage/blob/918d273bbb843bd09bb6c80d22d87c257a7e5343/plugins/techdocs-node/src/stages/generate/helpers.ts#L108

If you update this line then you should get that feature working with Harness 🚀

**Comment by OrkoHunter** [comment]
Thanks Andre for the suggestion! @cjlee01 can you please make the above change in a follow up PR? Thank you!

**Pull request merged**